app/views/candidate_interface/course_choices/review_enic_interruption/_not_needed.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="app-grid-column--grey">
+      <h1 class="govuk-heading-l"><%= t('review_enic_interruption.title') %></h1>
+
+      <p class="govuk-body"><%= t('review_enic_interruption.not_needed.more_likely_html') %></p>
+      <p class="govuk-body"><%= t('review_enic_interruption.not_needed.if_you_apply_enic_html', link: govuk_link_to(t('review_enic_interruption.not_needed.apply_for_enic_link'), candidate_interface_course_choices_course_review_path(current_application.id))) %></p>
```

Comment:
```suggestion
      <p class="govuk-body"><%= t('review_enic_interruption.not_needed.if_you_apply_enic_html', link: govuk_link_to(t('review_enic_interruption.not_needed.apply_for_enic_link'), candidate_interface_course_choices_course_review_path(@application_choice.id))) %></p>
```

This link is broken -- we want to review the application choice, not the application form.